### DISCUSSION TITLE
[Feature request] Faster GPU capability for zero shot yourTTS.

### DISCUSSION THREAD

**Discussion by t-naeem** [coqui-ai/TTS#2388] [General Q&A]
Hi, thanks for such an amazing work. I have a request for zero-shot yourTTS model. Since for zero shot capability, we need to train with at least a few thousand speakers according to this paper: https://www.mdpi.com/2079-9292/12/4/820

I see that zero shot yourTTS is trained on V100. It takes a lot of time to train a new language. I was wondering that if it can be adapted to a more powerful GPU so that the training time for a new language can be decreased. Thanks in advance.

**Comment by erogol** [comment]
zero-shot does not need training. You just provide a clip of the target speaker at inference and that's it.
